File: ejercicios/AMarotoChivite/AMarotoChivite-01-10-Catan-Java/src/main/java/gameCatan/utils/randomData.java
package gameCatan.utils;

import gameCatan.enums.TypeResource;

public class randomData {
    public static TypeResource randomType() {
        int random = (int) (Math.random() * 3 + 1);
        if (random == 1) {
            return TypeResource.WOOD;
        } else if (random == 2) {
            return TypeResource.COAL;
        } else {
            return TypeResource.SEED;
        }
    }
}

File: ejercicios/AMarotoChivite/AMarotoChivite-01-10-Catan-Kotlin/gameCatan/utils/randomData.kt
package gameCatan.utils

import gameCatan.enums.TypeResource

object randomData {
    @JvmStatic
    fun randomType(): TypeResource {
        val random = (Math.random() * 3 + 1).toInt()
        return if (random == 1) {
            TypeResource.WOOD
        } else if (random == 2) {
            TypeResource.COAL
        } else {
            TypeResource.SEED
        }
    }
}